src/components/Table.js

```diff
@@ -37,14 +37,16 @@ export default function UsersTable() {
     },
   ];
 
+  const memoColumns = useMemo(() => columns, []);
```

Comment:
Hi Juli, Its better to have the columns defined inside the memo, it would be something like

`const memoColumns = useMemo(() => ([
    { field: "id", headerName: "ID", flex: 1, minWidth: 25 },
    { field: "name", headerName: "NAME", flex: 2, minWidth: 150 },
    { field: "email", headerName: "EMAIL", flex: 2, minWidth: 150 },
    { field: "gender", headerName: "GENDER", flex: 2, minWidth: 150 },
    { field: "isActive", headerName: "STATUS", flex: 1, minWidth: 150 },
    {
      field: "actions",
      headerName: "ACTIONS",
      flex: 1,
      minWidth: 70,
      renderCell: (params) => {
        <>
          <GenericButton icon={VisibilityIcon}></GenericButton>
          <GenericButton icon={RemoveCircleIcon}></GenericButton>
        </>;
      },
    },
  ]), []);`

After that you can remove the columns variable